docker-compose.yml

```diff
@@ -0,0 +1,11 @@
+version: '3'
+
+services:
+  application:
+    container_name: template-node-ts
+    build:
+      context: .
+      dockerfile: Dockerfile.development
```

Comment:
There's already a Dockerfile for the src project, see my previous comment please :)